Urzadzenie_Oswietlenia/Urzadzenie_Oswietlenia/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace Urzadzenie_Oswietlenia
+{
+    class Program
+    {
+
+        private static readonly Socket _client = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+        private const int PORT = 100;
+        static void Main(string[] args)
+        {
+            LoopConnect();
+            SendLoop();
+            Exit(); 
+        }
+
+        private static void Exit()
+        {
+            _client.Shutdown(SocketShutdown.Both);
+            _client.Close();
+            Environment.Exit(0); 
+        }
+
+        private static void SendLoop()
+        {
+            while (true)
+            {
+                Console.WriteLine("Enter a request: ");
+                string req = Console.ReadLine();
```

Comment:
też fajnie jakby zmienna nie nazywała się `req` tylko `request` ;P dużo nie kosztuje

Reply:
i wgl to jakoś nie podoba mi się wczytywanie z konsoli, lepiej chyba było by argument do funkcji przekazywać, ale czaje, że to tylko pewnie na potrzeby pierwszego etapu